proto/lighthouse-response.proto

```diff
@@ -0,0 +1,363 @@
+    // There was an error while running the audit (check `error_message` for
+    // details). Score is NaN and should be ignored.
+    error = 6;
+  }
+
+  // The mode for how the score should be interpreted.
+  ScoreDisplayMode score_display_mode = 5;
+
+  // The human readable value that is displayed as the audit's result
+  repeated string display_value = 6;
+
+  // An explanation of audit-related issues encountered on the test page.
+  string explanation = 7;
+
+  // Error message from any exception thrown while running this audit.
+  string error_message = 8;
+
+  // Extra information provided by some types of audits.  Given as an arbitrary object.
+  google.protobuf.Struct details = 9;
+
+  // List of warnings associated with this audit
+  repeated string warnings = 10;
+}
+
+// Message containing the i18n data for the LHR - Version 1
+message I18nV1 {
+
+  // Message holding the formatted strings used in the renderer
+  message RendererFormattedStrings {
+    // The disclaimer shown below a performance metric value
+    string variance_disclaimer = 1;
+
+    // The heading for the estimated page load savings opportunity of an audit
+    string opportunity_resource_column_label = 2;
+
+    // The heading for the estimated page load savings of opportunitu audits
+    string opportunity_savings_column_label = 3;
+
+    // The error string shown next to an erroring audit
+    string error_missing_audit_info = 4;
+
+    // The label shown next to an audit or metric that has had an error
+    string error_label = 5;
+
+    // The label shown above a bulleted list of warnings
+    string warning_header = 6;
+
+    // The tooltip text on an expandable chevron icon
+    string audit_group_expand_tooltip = 7;
+
+    // The heading that is shown above a list of audits that are passing
+    string passed_audits_group_title = 8;
+
+    // The heading shown above a list of audits that do not apply to a page
+    string not_applicable_audits_group_title = 9;
+
+    // The heading shown above a list of audits that were not computerd in the run
+    string manual_audits_group_title = 10;
+
+    // The label shown preceding important warnings that may have invalidated an entire report
+    string toplevel_warnings_message = 11;
+
+    // The label that explains the score gauges scale (0-49, 50-89, 90-100)
+    string scorescale_label = 12;
+
+    // The label for values shown in the summary of critical request chains
+    string crc_longest_duration_label = 13;
+
+    // The label for the initial request in a critical request chain
+    string crc_initial_navigation = 14;
+
+    // The disclaimer shown under performance explaning that the network can vary
+    string ls_performance_category_description = 15;
+
+    // The title of the lab data performance category
+    string lab_data_title = 16;
+  }
+
+  // The message holding all formatted strings
+  RendererFormattedStrings renderer_formatted_strings = 1;
+
+  // DEPRECATED WITH USE OF LISTVALUES
```

Comment:
you might have to explain this one to me :)

the protobuf field is deprecated before we've even started using it? why not eliminate it now?

Reply:
Good question.  I wanted to talk about this.  Since `icuMessagePaths` is in the form:
```
"icuMessagePaths": {
    "key2" : [ {icuMessagePaths message}, {}, ...],
    "key2" : [ "string", "string", ... ],
...
```
i.e. it is irregular.  It makes it hard to put into a defined message like `IcuMessagePaths`.  So I punted and used `ListValue` which can take any objects in lists (even mixed objects like above).

So we could either:
1. modify `icuMessagePaths` to have lists of defined messages instead of sometimes strings, sometimes messages
2. keep using ListValues and keep the LHR the same but the proto definition is a little weaker

I kept the `icuMessagePaths` message in the proto file incase we want to use it.  If we decide to use ListValue, we can delete it.  It is just 1 of the 2 options.

Reply:
If we choose option 1 the `icuMessagePaths` objects in the LHR would look something like this then:
```json
"lighthouse-core/audits/byte-efficiency/uses-optimized-images.js | description": {
    "values": [
        {
            "path": "audits[uses-optimized-images].description"
        }
    ]
}, 
"lighthouse-core/lib/i18n/i18n.js | ms": {
    "values": [
        {
            "path": "audits[estimated-input-latency].displayValue", 
            "values": {
                "timeInMs": 16
            }
        }
    ]
}, 
```

Reply:
Talked with @brendankenny and @paulirish.  We decided to drop the `icuMessagePaths` field at this time since it is unused.